test/utils/ValidatorTester.js

```diff
@@ -63,6 +64,46 @@ const ValidatorTester = (requestEndpoint) => (location) => (field_name) => ({
         });
     },
 
+    mustBeFuture: () => {
+        test("should be a future Date", async () => {
+            const params = {
+                [field_name]: "1998-12-25",
+            };
+            const res = await requestEndpoint(params);
+
+            expect(res.status).toBe(422);
```

Comment:
Update to use the enum `HTTPStatus` from `http-status-codes`

Reply:
A rebase was necessary, as this was actually done in #63. However, took the chance and extracted some of the common assertions in `ValidatorTester.js` to a separate function, which makes the code cleaner imo :slightly_smiling_face:

Reply:
Please check it out now